README.md

```diff
@@ -1,45 +1,131 @@
-| [Bikeshed](https://tabatkins.github.io/bikeshed/) | Spec generator | [Tab Atkins Jr.](mailto:jackalmage@gmail.com) | [tabatkins/bikeshed](https://github.com/tabatkins/bikeshed) |
-| [https://httpwg.org/](https://httpwg.org/) | | | |
-| [https://krijnhoetmer.nl/irc-logs/](https://krijnhoetmer.nl/irc-logs/) | | | |
-| [https://w3ctag.github.io/](https://w3ctag.github.io/) | | | |
-| [https://w3ctag.org/](https://w3ctag.org/) | | | |
-| [https://tag.w3.org/](https://tag.w3.org/) | | | |
-| [https://extensiblewebreportcard.org/](https://extensiblewebreportcard.org/) | | | |
-| [https://wiki.css-houdini.org/](https://wiki.css-houdini.org/) | | | |
-| [https://hg.css-houdini.org/drafts](https://hg.css-houdini.org/drafts) | | | |
-| [https://drafts.css-houdini.org](https://drafts.css-houdini.org) | | | |
-| [https://drafts.htmlwg.org/](https://drafts.htmlwg.org/) | | | |
-| [https://specs.webplatform.org/](https://specs.webplatform.org/) | | | |
-| [https://specs.webplatform.org/docs/](https://specs.webplatform.org/docs/) | | | |
-| [https://github.com/webspecs/docs](https://github.com/webspecs/docs) | | | |
-| [https://discourse.specifiction.org/](https://discourse.specifiction.org/) | | | |
-| [https://fxtf.org/](https://fxtf.org/) | | | |
-| [https://hg.fxtf.org/](https://hg.fxtf.org/) | | | |
-| [https://dir.w3.org/](https://dir.w3.org/) | | | |
-| [https://www.html-tidy.org/](https://www.html-tidy.org/) | | | |
-| [https://www.htacg.org/](https://www.htacg.org/) | | | |
-| [https://github.com/htacg/](https://github.com/htacg/) | | | |
-| [https://discourse.wicg.io/](https://discourse.wicg.io/) | | | |
-| [https://validator.github.io/validator/](https://validator.github.io/validator/) | | | |
-| [https://github.com/WICG](https://github.com/WICG) | | | |
-| [https://indieweb.org/](https://indieweb.org/) | | | |
-| [https://websub.rocks/](https://websub.rocks/) | | | |
-| [https://webmention.rocks/](https://webmention.rocks/) | | | |
-| [https://webmention.net/](https://webmention.net/) | | | |
-| [https://wpt.fyi/](https://wpt.fyi/) | | | |
-| [https://web-platform-tests.org/](https://web-platform-tests.org/) | | | |
-| [https://responsiveimages.org/](https://responsiveimages.org/) | | | |
-| [https://wicg.io/](https://wicg.io/) | | | |
-| [https://testthewebforward.org/](https://testthewebforward.org/) | | | |
-| [https://credweb.org/](https://credweb.org/) | | | |
-| [https://browsers.evolt.org/browsers/archive/worldwideweb/NeXT/](https://browsers.evolt.org/browsers/archive/worldwideweb/NeXT/) | | | |
+# List of W3C-related servers/services
+
+The inventory below lists all the domains and GitHub organizations used by the W3C community but not operated by W3C. The owners are welcome to provide details on the infrastructure and maintainance of their servers.
+
+## [css-houdini.org](https://css-houdini.org/)
+Purpose:
+Owners:
+Comments:
```

Comment:
This will actually be displayed on a single line (see: https://github.com/w3c/related-tooling/blob/servers-only-1/README.md), you might want to add new lines (to have different paragraphs)  or make this a list.

Reply:
resolved in https://github.com/w3c/related-tooling/pull/2/commits/8cc9efe2398cf6f149666c66d2955423af860ce0